src/Http/Controllers/Character/MarketController.php

```diff
@@ -25,7 +25,7 @@
 use Seat\Services\Repositories\Character\Market;
```

Comment:
# Character Market
- sorting by type (sell/buy) is throwing an exception
```
DataTables warning: table id=character-market - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'bid' in 'order clause' (SQL: select 

                --

                -- Select All

                --

                *,



                --

                -- Start stationName Lookup

                --

                CASE

                when a.location_id BETWEEN 66015148 AND 66015151 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id-6000000)

                when a.location_id BETWEEN 66000000 AND 66014933 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id-6000001)

                when a.location_id BETWEEN 66014934 AND 67999999 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id-6000000)

                when a.location_id BETWEEN 60014861 AND 60014928 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id)

                when a.location_id BETWEEN 60000000 AND 61000000 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id)

                when a.location_id >= 61000000 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id)

                else (SELECT m.itemName FROM mapDenormalize AS m

                    WHERE m.itemID = a.location_id) end

                    AS stationName from character_orders as a inner join `invTypes` on `a`.`type_id` = `invTypes`.`typeID` inner join `invGroups` on `invTypes`.`groupID` = `invGroups`.`groupID` where `a`.`character_id` = 90795931 order by `bid` asc limit 10 offset 0)
```

- sorting by volume is throwing an exception
```
DataTables warning: table id=character-market - Exception Message:

		SQLSTATE[42S22]: Column not found: 1054 Unknown column 'volEntered' in 'order clause' (SQL: select 

                --

                -- Select All

                --

                *,



                --

                -- Start stationName Lookup

                --

                CASE

                when a.location_id BETWEEN 66015148 AND 66015151 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id-6000000)

                when a.location_id BETWEEN 66000000 AND 66014933 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id-6000001)

                when a.location_id BETWEEN 66014934 AND 67999999 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id-6000000)

                when a.location_id BETWEEN 60014861 AND 60014928 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id)

                when a.location_id BETWEEN 60000000 AND 61000000 then

                    (SELECT s.stationName FROM staStations AS s

                      WHERE s.stationID = a.location_id)

                when a.location_id >= 61000000 then

                    (SELECT d.name FROM `sovereignty_structures` AS c

                      JOIN universe_stations d ON c.structure_id = d.station_id

                      WHERE c.structure_id = a.location_id)

                else (SELECT m.itemName FROM mapDenormalize AS m

                    WHERE m.itemID = a.location_id) end

                    AS stationName from character_orders as a inner join `invTypes` on `a`.`type_id` = `invTypes`.`typeID` inner join `invGroups` on `invTypes`.`groupID` = `invGroups`.`groupID` where `a`.`character_id` = 90795931 order by `volEntered` asc limit 10 offset 0)
```

- making a research is throwing an exception
```
DataTables warning: table id=character-market - Exception Message:

			SQLSTATE[42S22]: Column not found: 1054 Unknown column 'bid' in 'where clause' (SQL: select count(*) as aggregate from (select '1' as `row_count` from character_orders as a inner join `invTypes` on `a`.`type_id` = `invTypes`.`typeID` inner join `invGroups` on `invTypes`.`groupID` = `invGroups`.`groupID` where `a`.`character_id` = 90795931 and (LOWER(`issued`) LIKE %rf% or LOWER(`bid`) LIKE %rf% or LOWER(`volEntered`) LIKE %rf% or LOWER(`price`) LIKE %rf% or LOWER(`price`) LIKE %rf% or LOWER(`typeName`) LIKE %rf%)) count_row_table)
```